source/rest/manager/lib/Api/AbstractConfigApi.php

```diff
@@ -1,17 +1,15 @@
 <?php
+
 /**
- * AbstractConfigApi
- *
- * PHP version 7
+ * CometVisu Manager backend
+ * PHP version 7.4
```

Comment:
bump to PHP 8.0?
Is it relevant here or is the central definition of PHP >= 8.0 sufficient?

Reply:
This is a comment in generated code, changing it manually would get lost on every new generator execution.